app/views/layouts/application.html.slim

```diff
@@ -0,0 +1,33 @@
+doctype html
+html
+  head
+    title
+      | OSBB
+    = csrf_meta_tags
+    = csp_meta_tag
+    = stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload'
+    = javascript_pack_tag 'application', 'data-turbolinks-track': 'reload'
+
```

Comment:
maybe move it to 'partials/_header.html.slim' and render header file

Reply:
Done